acceptance/api/v1/application_update_test.go

```diff
@@ -53,13 +53,15 @@ var _ = Describe("AppUpdate Endpoint", LApplication, func() {
 			env.MakeContainerImageApp(app, 1, containerImageURL)
 			defer env.DeleteApp(app)
 
-			appObj := appFromAPI(namespace, app)
+			appObj := appShow(namespace, app)
 			Expect(appObj.Workload.Status).To(Equal("1/1"))
 
-			status, _ := updateAppInstances(namespace, app, 3)
-			Expect(status).To(Equal(http.StatusOK))
+			request := map[string]interface{}{"instances": 3}
+			_, statusCode := appUpdate(namespace, app, toJSON(request))
+			Expect(statusCode).To(Equal(http.StatusOK))
 
 			Eventually(func() string {
+				return appShow(namespace, app).Workload.Status
 				return appFromAPI(namespace, app).Workload.Status
```

Comment:
```
  Running [/home/runner/golangci-lint-1.52.2-linux-amd64/golangci-lint run --out-format=github-actions --timeout=10m --skip-files docs.go] in [] ...
  Error: : # github.com/epinio/epinio/acceptance/api/v1_test [github.com/epinio/epinio/acceptance/api/v1.test]
  Error: acceptance/api/v1/application_update_test.go:65:12: undefined: appFromAPI (typecheck)
```

Reply:
This happens when you try to stage lines one by one :disappointed: 

Fixed!